public/challenge-contributions/1/asavovStyles.css

```diff
@@ -0,0 +1,56 @@
+html {
```

Comment:
The CSS should be in the HTML file.

![image](https://github.com/rumenpetrov/fe-hunger-games/assets/7723928/8e055607-2d72-4065-bccb-ca06b8efd275)